api/client/src/intTest/java/org/projectnessie/client/auth/oauth2/ITOAuth2Client.java

```diff
@@ -168,6 +172,7 @@ void testOAuth2ClientWithBackgroundRefresh() throws Exception {
               () -> {
                 tryUseAccessToken(validatingClient, client1.getCurrentTokens().getAccessToken());
                 tryUseAccessToken(validatingClient, client2.getCurrentTokens().getAccessToken());
+                tryUseAccessToken(validatingClient, client3.getCurrentTokens().getAccessToken());
```

Comment:
This was an embarrassing "whoops!" moment :-)

Reply:
`client3` was introduced a while ago, but it wasn't being tested because this line was missing.